node/subsystem-util/src/runtime/mod.rs

```diff
@@ -72,7 +75,7 @@ pub struct ValidatorInfo {
 
 impl RuntimeInfo {
 	/// Create a new `RuntimeInfo` for convenient runtime fetches.
-	pub fn new(keystore: SyncCryptoStorePtr) -> Self {
+	pub fn new(keystore: Option<SyncCryptoStorePtr>) -> Self {
```

Comment:
I don't quite see where the keystore is ever `None` :see_no_evil:

Reply:
In the collator protocol (collator side) we don't need it, because we are not a validator.